### ISSUE TITLE
Button size refinements

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#426]
Buttons should have a consistent height, and a minimum width that is equal to that height.

Updates expected for this issue:

- ghost/action/icon buttons are the same height as regular buttons labeled with text
- ghost/action/icon buttons are equal in height and width (this might be resolved by #330, but should be verified as part of that)

**Comment by jgiardino** [comment]
The [AAA requirement for target size is 44x44 px](https://www.w3.org/WAI/WCAG21/quickref/#target-size). This item is currently listed in our guideline, but since our goal is AA, we could remove it and replace it with a PF specific guideline. In chatting with @kybaker, the minimum size should be 32x32 px.

I've added this as an update to make as part of #403
